aa-sdk/ethers/src/__tests__/provider-adapter.test.ts

```diff
@@ -19,13 +26,26 @@ describe("Simple Account Tests", async () => {
 
   it("should correctly sign the message", async () => {
     const provider = await givenConnectedProvider({ alchemyProvider, signer });
+    const message = "hello world";
+    // todo: why does this provider's version of signMessage return a string, instead of Hex?
```

Comment:
Seems to be because this provider is an ethers provider rather than viem because we're in the ethers unit tests, and ethers doesn't distinguish hex strings.

Reply:
makes sense, removed comment.